rpc/rpc-wrappers.go

```diff
@@ -504,6 +505,39 @@ func (vac ValidationAuthorityClient) CheckCAARecords(ident core.AcmeIdentifier)
 	return
 }
 
+func NewPublisherAuthorityServer(rpc RPCServer, impl core.PublisherAuthority) (err error) {
+	rpc.Handle(MethodSubmitToCT, func(req []byte) (response []byte, err error) {
+		cert, err := x509.ParseCertificate(req)
+		if err != nil {
+			// AUDIT[ Improper Messages ] 0786b6f2-91ca-4f48-9883-842a19084c64
+			improperMessage(MethodIssueCertificate, err, req)
+			return
+		}
+
+		err = impl.SubmitToCT(cert)
+		return
+	})
+
+	return nil
+}
+
+// PublisherAuthorityClient is a client to communicate with the Publisher Authority
+type PublisherAuthorityClient struct {
+	rpc RPCClient
+}
+
+// NewPublisherAuthorityClient constructs an RPC client
+func NewPublisherAuthorityClient(client RPCClient) (pub PublisherAuthorityClient, err error) {
+	pub = PublisherAuthorityClient{rpc: client}
+	return
+}
+
+// SubmitToCT sends a request to submit a certifcate to CT logs
+func (pub PublisherAuthorityClient) SubmitToCT(cert *x509.Certificate) (err error) {
+	_, err = pub.rpc.DispatchSync(MethodSubmitToCT, cert.Raw)
```

Comment:
Not for the current implementation, in the future we will probably want either another external tool (or to integrate the functionality into one of the existing ones) that can scan the database for certificates that haven't been submitted to all configured logs and attempt to re-submit them. This tool would probably want it to be synchronous so that it can properly retrieve any error messages that are passed back.